### DISCUSSION TITLE
Default value from shell command output

### DISCUSSION THREAD

**Discussion by Vayel** [copier-org/copier#913] [Q&A]
Hi!

I have a `email` question and I'd like to use the output of `git config --get user.email` as default value. Is that possible?

Thanks!
Vincent

**Comment by yajo** [comment]
https://pypi.org/project/jinja2-shell-extension/ can fill your purpose. See https://copier.readthedocs.io/en/v7.0.1/configuring/#jinja_extensions to know how to enable extensions in a template.
